src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.Scanner;
+import java.util.Arrays;
 public class Duke {
+
+    private static Scanner in = new Scanner(System.in);
+    private static final Task[] tasks = new Task[100];
+
+    public static void command(String line) {
+        String exitCommand = "bye";
+        String toDo = "list";
+        String checkAsDone = "mark";
```

Comment:
You might want to consider converting them into named constants instead of a variables since it is unlikely for them to be changed.
E.g. `public static final String EXIT_COMMAND = "bye";`